File: ImagePickerMaster/src/main/java/com/example/apple/glidetest/utils/OsUtils.java
package com.example.apple.glidetest.utils;

import android.content.Context;
import android.util.DisplayMetrics;
import android.view.View;

/**
 * Created by Apple on 17/5/31.
 */

public class OsUtils {
    public static int dp2px(Context context, float dpValue) {
        final float scale = context.getResources().getDisplayMetrics().density;
        return (int) (dpValue * scale + 0.5);
    }

    public static int getScreenHeight(Context context) {
        DisplayMetrics displayMetrics = context.getResources().getDisplayMetrics();
        return displayMetrics.heightPixels;
    }

    public static boolean isVisible(View view) {
        return view.getVisibility() == View.VISIBLE;
    }

    public static int getStatusBarHeight(Context context) {
        int statusBarHeight = -1;
        int resourceId = context.getResources().getIdentifier("status_bar_height", "dimen", "android");
        if (resourceId > 0) {
            statusBarHeight = context.getResources().getDimensionPixelSize(resourceId);
        }
        return statusBarHeight;
    }
}

File: ImagePickerMaster/src/main/java/com/example/apple/glidetest/utils/OsUtils.kt
package com.example.apple.glidetest.utils

/**
 * Created by Apple on 17/5/31.
 */

object OsUtils {
    fun getIndexInList(list: List<String>, str: String): Int {
        if (list.isNotEmpty()) {
            for (i in list.indices) {
                if (list[i] == str) {
                    return i
                }
            }
        }
        return -1
    }
}
